src/stk/molecular/topology_graphs/metal_complex/paddlewheel/paddlewheel.py

```diff
@@ -0,0 +1,93 @@
+"""
+Paddlewheel
+===========
+
+"""
+
+from ..metal_complex import MetalComplex
+from ..vertices import _MetalVertex, _BiDentateLigandVertex
+from ...topology_graph import Edge
+
+
+class Paddlewheel(MetalComplex):
+    """
+    Represents a metal complex topology graph.
+
+    Metal building blocks with at least four functional groups are
+    required for this topology.
+
+    Ligand building blocks with two functional groups are required for
+    this topology.
+
+    :class:`.BuildingBlock` placements:
+        metals: (0, 1)
+        ligands: (0, 1, 2, 3)
+
+    See :class:`.MetalComplex` for more details and examples.
+
+    """
+
+    _metal_vertex_prototypes = (
+        _MetalVertex(0, [0, 1, 0]),
+        _MetalVertex(1, [0, -1, 0]),
+    )
+    _ligand_vertex_prototypes = (
+        _BiDentateLigandVertex(2, [2, 0, 0]),
+        _BiDentateLigandVertex(3, [0, 0, 2]),
+        _BiDentateLigandVertex(4, [-2, 0, 0]),
+        _BiDentateLigandVertex(5, [0, 0, -2]),
+    )
+
```

Comment:
You should be able to initialize this topology as `stk.metal_complex.Paddlewheel(metal, ligand)`, ie where the user gives a single metal and a single ligand, instead of a dict mapping.

Reply:
Do you want this for all topologies?